umake/frameworks/ide.py

```diff
@@ -110,24 +106,23 @@ def done(download_result):
 
             logger.debug("Preparing to download the main archive.")
             if arch == 'i686':
-                download_url = self.DOWNLOAD_URL_PAT.format(arch='', suf='')
+                download_url = self.download_page_url.format(arch='', suf='')
             elif arch == 'x86_64':
-                download_url = self.DOWNLOAD_URL_PAT.format(arch='-x86_64',
-                                                            suf='')
+                download_url = self.download_page_url.format(arch='-x86_64',
```

Comment:
That's a good one. Here for the commit separation, I would think:
- one commit introducing BaseEclipse, adding the new property + the rename
and that's it! :)

This kind of change can be in the "add Eclipse Mars" PR, as it's directly related to it.